x/stakeibc/types/codec.go

```diff
@@ -22,6 +22,7 @@ func RegisterCodec(cdc *codec.LegacyAmino) {
 	cdc.RegisterConcrete(&AddValidatorsProposal{}, "stakeibc/AddValidatorsProposal", nil)
 	cdc.RegisterConcrete(&ToggleLSMProposal{}, "stakeibc/ToggleLSMProposal", nil)
 	cdc.RegisterConcrete(&MsgRestoreInterchainAccount{}, "stakeibc/RestoreInterchainAccount", nil)
+	cdc.RegisterConcrete(&MsgCloseDelegationChannel{}, "stakeibc/CloseDelegationChannel", nil)
```

Comment:
Does this allow us to sign with Ledger / did we test with Ledger?

Reply:
[this](https://github.com/Stride-Labs/stride/pull/1242/files#diff-588e281d6fae46aa27bd0a5a2a3cafcd412244b49fe96977892590da86095cb9R16) should be what allows us to sign with ledger. Haven't test it

Reply:
I thought we need to register a legacy amino codec [like this](https://github.com/Stride-Labs/stride/blob/3e1e5db9b555e13cd6ab5fcf36220fb3e0e2c856/x/airdrop/types/codec.go#L15)

Reply:
yeah i guess it's both - but the issue we've had in the past is that we didn't implement the legacy interface

Reply:
sorry disregard - I see this _is_ the LegacyAmino registration